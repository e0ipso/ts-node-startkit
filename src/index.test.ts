import index from './index';

describe('index', () => {
  it('should return the expected string', async () => {
    const res = await index();
    expect(res).toMatchSnapshot();
  });
});
