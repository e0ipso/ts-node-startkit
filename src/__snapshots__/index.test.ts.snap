// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should return the expected string 1`] = `
"import { readFile } from 'fs';
import { promisify } from 'util';

const readFileP = promisify(readFile);

/**
 * IMPORTANT NOTE
 * This is a dummy file that should be deleted in your forked project.
 * The file exists so the CI does not choke on the \`tsc\` compilation.
 */
export default async function (): Promise<string> {
  const data = await readFileP(__filename);
  return data.toString();
}
"
`;
